src/Sylius/Bundle/WebBundle/Resources/views/Backend/Inventory/macros.html.twig

```diff
@@ -72,7 +72,7 @@
                                     <td>
                                         <ul>
                                         {% for option in variant.options %}
-                                            <li><strong>{{ option.name }}</strong>: {{ option.value }}</li>
+                                            <li><strong>{{ option.presentation }}</strong>: {{ option.value }}</li>
```

Comment:
No problem, But there are more places of confusion like this I would say. 
* all options, which in real are optionValue should be renamed in twig and other places.  `{% for optionValue in variant.options %}`. wdyt?
* rename the method `getPresentation` to `getOptionName` which would be consitent with `getOptionCode`  already in the object, and resolve confusion of what you can expect from presentation method.

wdyt?